keras_nlp/src/models/backbone.py

```diff
@@ -133,6 +133,9 @@ def get_config(self):
                         policy_map[layer.path] = layer.dtype_policy
                 if len(policy_map) > 0:
                     config.update({"dtype": policy_map})
+        # Before Keras 3.2, there is no `keras.dtype_policies.get`.
+        except AttributeError:
+            pass
```

Comment:
I was wondering why we don't handle this case like the first one using `keras.DTypePolicy(dtype)`.

Reply:
nvm, I just noticed they are completely different cases!